scripts/shp_to_geojson.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Convert geo files from .shp to .geojson format.
+# This script depends on the `ogr2ogr` library from `gdal`.
+# Install it from homebrew via `brew install gdal`.
+
+ogr2ogr -f GeoJSON ~/tmp/polling_places/MA_025_Statewide_Precincts_Final/MA_025_Statwide2021PrecinctsFinal.geojson ~/tmp/polling_places/MA_025_Statewide_Precincts_Final/MA_025_Statwide2021PrecinctsFinal.shp
```

Comment:
I feel kinda weird about hard-coded paths like this but can't think of a better resolution.

Reply:
Yeah, it could be a full-fledged script that takes arguments, but it felt more like "documentation" than something that needed a lot of time spent on in.